### DISCUSSION TITLE
alldata

### DISCUSSION THREAD

**Discussion by TomAff** [moneymanagerex/moneymanagerex#5375] [Q&A]
Is there an update for the view "alldata" for DB ver.17?

**Comment by TomAff** [comment]
Done. No answer needed

**Reply by omalleypat** [reply]
@TomAff can you share the sql for the alldata view update?

**Reply by omalleypat** [reply]
In searching more, I found this: https://github.com/moneymanagerex/moneymanagerex/issues/5724
